db/migrate/20250918151250_add_uplift_fields_to_teachers.rb

```diff
@@ -0,0 +1,8 @@
+class AddUpliftFieldsToTeachers < ActiveRecord::Migration[8.0]
+  def change
+    change_table :teachers, bulk: true do |t|
+      t.boolean :pupil_premium_uplift, default: false, null: false
```

Comment:
can those both be prefixed by `ect` as we aren't adding for mentors?

Reply:
prefixed columns with `ect`